org-super-agenda.info

```diff
@@ -1,4 +1,4 @@
-This is README.info, produced by makeinfo version 5.2 from README.texi.
+This is README.info, produced by makeinfo version 6.7 from README.texi.
```

Comment:
I appreciate your being through, but if you don't mind, please omit the updates to the Info manual from the PR, and I can regenerate it myself using the same makeinfo version, to minimize differences.

Reply:
done